.github/workflows/swiftlint_analyze.yml

```diff
@@ -0,0 +1,33 @@
+name: SwiftLint Analyze
+
+on:
+  push:
+    branches: [master]
+    paths: 
+      - '.github/workflows/swiftlint_analyze.yml'
+      - 'Yams.xcodeproj/**'
+      - 'Sources/**'
+      - 'Tests/**'
+      - '!Tests/LinuxMain.swift'
+  pull_request:
+    paths:
+      - '.github/workflows/swiftlint_analyze.yml'
+      - 'Yams.xcodeproj/**'
+      - 'Sources/**'
+      - 'Tests/**'
+      - '!Tests/LinuxMain.swift'
+
+jobs:
+  Analyze:
+    runs-on: macOS-10.14
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_11.2.app
+    steps:
+      - uses: actions/checkout@v1
+      - name: Generate xcodebuild.log
+        run: xcodebuild -scheme Yams -project Yams.xcodeproj clean build-for-testing > xcodebuild.log
+        shell: bash
+      - name: Install SwiftLint
+        run: HOMEBREW_NO_AUTO_UPDATE=1 brew install https://raw.github.com/Homebrew/homebrew-core/master/Formula/swiftlint.rb
```

Comment:
I think we can hardcode to use 0.36.0 now.

Reply:
Do you think we should pin the version of SwiftLint here?